src/OpenDB/test/data/basic_read.blif

```diff
@@ -0,0 +1,8 @@
+.model tmp_circuit
```

Comment:
why are the tests in opendb?
also 3_place.def is way bigger than you should have in a UNIT TEST